test/models/highlight_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class HighlightTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
if there is a `spec` directory, there is no need to also have a `test` directory